.github/workflows/release.yml

```diff
@@ -0,0 +1,86 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+      - dev-*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-binaries:
+    runs-on: ubuntu-latest
+    if: github.repository == 'numaproj-labs/numaplane'
+    name: Build binaries
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+
+  build-push-linux-multi:
+    name: Build & push linux/amd64 and linux/arm64
+    needs: [ build-binaries ]
+    runs-on: ubuntu-latest
+    if: github.repository == 'numaproj-labs/numaplane'
+    strategy:
+      matrix:
+        target: [ numaplane ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Registry Login
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAYIO_USERNAME }}
+          password: ${{ secrets.QUAYIO_PASSWORD }}
+
+      - name: set Version
+        id: version
+        run: |
+          tag=$(basename $GITHUB_REF)
+          if [ $tag = "main" ]; then
+            tag="latest"
+          fi
+          echo "VERSION=$tag" >> $GITHUB_OUTPUT
+      - name: Container build and push with arm64/amd64
+        run: make image docker-push IMAGE_FULL_PATH=${{ secrets.QUAYIO_ORG }}/numaplane:${{ steps.version.outputs.VERSION }} CONTAINER_TOOL=docker
```

Comment:
image name should be [numaplane-controller](https://github.com/numaproj-labs/numaplane/blob/main/Makefile#L7) - did you get the `numaplane` image name from somewhere else?

Reply:
if i change the line in the release file from `numproj/numaplane` to `numproj/numaplane-controller`  it will not push because `https://quay.io/repository/numaproj/numaplane-controller` does not exist